esphome/components/tsl2591/tsl2591.h

```diff
@@ -224,7 +224,7 @@ class TSL2591Component : public PollingComponent, public i2c::I2CDevice {
   float get_setup_priority() const override;
 
  protected:
-  const char *name_;  // TODO: extend esphome::Nameable
+  const char *name_;  // TODO: extend esphome::EntityBase
```

Comment:
```suggestion
  const char *name_;
```
This component really isn't an entity, so that's not necessary (the name only is used for logging).

Reply:
Thanks, I was wondering about this one... figured I'd better leave the comment, updated to reflect that Nameable is no longer a thing.